sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventController.java

```diff
@@ -756,6 +758,13 @@ public CommitDiscardWrapperComponent<EventDataForm> getEventDataEditComponent(fi
 			editView.getButtonsPanel().addComponentAsFirst(new AutomaticDeletionLabel(automaticDeletionInfoDto));
 		}
 
+		if (event.isDeleted()) {
+			editView.getWrappedComponent().getField(EventDto.DELETE_REASON).setVisible(true);
+			if (editView.getWrappedComponent().getField(EventDto.DELETE_REASON).getValue()== DeleteReason.OTHER_REASON){
```

Comment:
see above (please check all Controllers for formatting)

Reply:
verified all Controllers, no formatting issues found